platform/windows/context_gl_windows.cpp

```diff
@@ -63,16 +66,76 @@ int ContextGL_Windows::get_window_height() {
 	return OS::get_singleton()->get_video_mode().height;
 }
 
+bool ContextGL_Windows::should_vsync_via_compositor() {
+
+	if (OS::get_singleton()->is_window_fullscreen()) {
+		return false;
+	} else {
```

Comment:
No need for the `else` and extra indentation since we're just returning in the previous branch.

Reply:
This has been fixed.